github/ipld.yml

```diff
@@ -1671,6 +1678,33 @@ teams:
       member:
         - RangerMauve
     privacy: closed
+  github-mgmt stewards:
+    # NOTE: created to capture users with push+ access to github-mgmt repository
+    #  using a team instead of direct collaborators because we want to reference it in the CODEOWNERS file
+    create_default_maintainer: false
+    description: Users that are effectively org admins
+    members:
+      # WARN: membership here should be treated exactly as cautiosly as having an org admin role
+      # ATTN: members are expected to:
+      #  - be familiar with GitHub Management
+      #  - be ready to triage/review org configuration change request in github-mgmt
+      maintainer:
+        - aschmahmann
+        - rvagg
+      member:
+        - BigLep
+        - lidel
+        - RangerMauve
```

Comment:
```suggestion
```

Removing from now until get a sense of whether they even want to have this kind of permissions.  IPLD org doesn't have as much movement either so having more people isn't as critical at the moment.